src/app/CommandResponseSender.cpp

```diff
@@ -226,6 +230,21 @@ void CommandResponseSender::OnInvokeCommandRequest(Messaging::ExchangeContext *
     }
 }
 
+size_t CommandResponseSender::GetCommandMaxBufferSize()
+{
+    VerifyOrExit(mExchangeCtx && mExchangeCtx->HasSessionHandle(),
```

Comment:
This would probably more readable as an actual if/log/return without the weird exit bits.

Reply:
Ok. Changed to an if-return.